src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1957,72 +1957,111 @@ limitations under the License.
             <include cluster="Low Power" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="Wake on LAN" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="Target Navigator" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="Account Login" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Content Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="Content Launcher" client="false" server="true" clientLocked="false" serverLocked="true"></include>
         </clusters>
     </deviceType>
-
     <deviceType>
         <name>MA-basic-videoplayer</name>
         <domain>CHIP</domain>
         <typeName>Matter Basic Video Player</typeName>
         <profileId editable="false">0x0103</profileId>
         <deviceId editable="false">0x0028</deviceId>
         <clusters lockOthers="true">
-            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
-                <requireAttribute>IDENTIFY_TIME</requireAttribute>
-                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
-                <requireCommand>Identify</requireCommand>
-                <requireCommand>IdentifyQuery</requireCommand>
-                <requireCommand>TriggerEffect</requireCommand>
-            </include>
             <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
                 <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
                 <requireAttribute>SERVER_LIST</requireAttribute>
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
-                <requireAttribute>BINDING</requireAttribute>
-            </include>
             <include cluster="Media Playback" client="false" server="true" clientLocked="false" serverLocked="true"></include>
             <include cluster="Keypad Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
-            <include cluster="Application Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Media Input" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="Media Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
             <include cluster="On/Off" client="false" server="true" clientLocked="false" serverLocked="false"></include>
```

Comment:
No, this is wrong.  On/Off is mandatory, Media Input is optional.  Please look at what my actual review comments were last time....

Reply:
Sorry, I've had a conflict with this part of the code and I've chosen the line that overriden your commit suggestion.

Updated no to match the suggested version.